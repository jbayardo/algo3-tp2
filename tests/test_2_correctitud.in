10 20
0 0 10 20; 0 2 10 20; 0 3 10 20; 0 5 10 20; 0 20 10 20
9 18
0 5 3 8; 3 5 3 1; 3 0 4 3; 3 18 4 11; 4 6 5 3; 5 1 6 2; 5 4 9 3
6 9
0 6 2 3; 1 8 3 8; 3 4 6 9; 2 1 4 5; 4 7 6 0
6 9
0 6 2 3; 0 9 1 8; 1 8 3 8; 3 4 6 9; 2 1 4 5; 4 7 6 0
6 9
0 6 2 3; 0 9 1 8; 1 8 3 8; 2 1 4 5; 4 7 6 0
3 17
0 1 0 13; 0 3 1 0; 0 8 1 5; 0 12 1 7; 0 14 1 9; 1 1 2 1; 1 6 2 3; 1 8 2 9; 1 10 2 13; 2 2 3 0; 2 4 3 1; 2 10 3 2; 2 14 3 16