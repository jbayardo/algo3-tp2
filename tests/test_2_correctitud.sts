Excercise 2 Generate Graph Timer			48,35,13,13,13,14,13
Excercise 2 Timer			8,21,8,11,12,13,9
