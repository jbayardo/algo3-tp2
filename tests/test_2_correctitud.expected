2
38
25
19
25
13